### ISSUE TITLE
BUG/PROBLEM: KTOR Client does not recive custom HTTP status code

### ISSUE THREAD

**Issue by CayzerOK** [ktorio/ktor#622]
Hallo!

I have REST service with custom HTTP Status Codes. It looks like "400, Invalid Email"

But ktor client does not response that. It show me standart code description ("400, Bad Request")

**Comment by e5l** [comment]
Hi, @CayzerOK. Thanks for the report.
What engine do you use?

**Comment by CayzerOK** [comment]
server and client using ktor.

Generic call fun:

```
fun Route.LoginUser() {
    post<lLoginData> { loginCall ->
        if(!isEmailValid(loginCall.email)) {call.respond(HttpStatusCode(400, "Invalid Email"))}
        else call.sessions.set(SQL.Login(loginCall.email,loginCall.password))
        call.respond(HttpStatusCode.OK)
    }
}
```

**Comment by e5l** [comment]
Fixed in `1.0.0-alpha-3`
